sdk/authorization/arm-authorization/README.md

```diff
@@ -26,21 +26,22 @@ npm install @azure/ms-rest-nodeauth
 ##### Sample code
 
 ```typescript
```

Comment:
Should this change to javascript?

Reply:
Yes, fixed with https://github.com/Azure/azure-sdk-for-js/pull/10356/commits/3f9d6af6bc03808963581d46055046a7290d1665